internal/provider-integration/command.go

```diff
@@ -0,0 +1,40 @@
+package provider_integration
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+)
+
+type command struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type providerIntegrationOut struct {
+	Id                 string   `human:"ID" serialized:"id"`
+	Name               string   `human:"Name" serialized:"name"`
+	Provider           string   `human:"Provider" serialized:"provider"`
+	Environment        string   `human:"Environment" serialized:"environment"`
+	IamRoleArn         string   `human:"IAM Role Arn" serialized:"iam_role_arn"`
+	ExternalId         string   `human:"External ID" serialized:"external_id"`
+	CustomerIamRoleArn string   `human:"Customer IAM Role Arn" serialized:"customer_iam_role_arn"`
+	Usages             []string `human:"Usages" serialized:"usages"`
+}
+
+func New(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:         "provider-integration",
+		Aliases:     []string{"pi"},
+		Short:       "Manage CLI provider integrations.",
```

Comment:
Are these provider integrations only stored in the CLI? Maybe "Confluent Cloud provider integrations" is the proper terminology here.